roadmap/implementers-guide/src/types/approval.md

```diff
@@ -0,0 +1,128 @@
+# Approval Types
+
+## ApprovalId
+
+The public key of a keypair used by a validator for approval voting.
+
+## AssignmentId
+
+The private key of a keypair used by a validator for approval voting.
+
+## AssignmentCert
+
+An `AssignmentCert`, short for Assignment Certificate, is a piece of data provided by a validator to prove that they have been selected to perform secondary approval checks on an included candidate.
```

Comment:
I'm actually okay with the name certificate here..  It's initially confusing since a certificate is normally a signature by one key on another key, but in this case it's a VRF by a node's own key that delegates them the authority to sign as an approval checker.  I'll let you know if I think of anything better, but let's keep the name cert for now.

Reply:
Let me know what you come up with! I couldn't think of anything good either.